### PR TITLE
Add dashboard routing base on role

### PR DESCRIPTION
**Pull request by Pursain**

### CODE REVIEW

File: src/hooks/useRole.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
+import userService from '../user/userService';
+
+const useRole = () => {
+  const [role, setRole] = useState(null);
```
Comment (appsicle) [inline src/hooks/useRole.js:5]: i have been using undefined as my default state, idk whats better

File: src/common/Dropdown.js [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
 function Dropdown({ name, profileURL, onLogoutSuccess }) {
   const dropdownRef = useRef(null);
   const [isActive, setIsActive] = useDetectOutsideClick(dropdownRef, false);
   const onClick = () => setIsActive(!isActive);
 
+  const { role, roleError } = useRole();
+  const [dashboardLink, setDashboardLink] = useState('/');
+
+  useEffect(() => {
+    switch (role) {
```
Comment (appsicle) [inline src/common/Dropdown.js:19]: could this be cleaner as a ternary ? operator

Comment (Pursain) [reply src/common/Dropdown.js:19]: eslint wont let me do 3 way ternary :(((((((((

File: src/hooks/useRole.js (hunk 1 above)
Comment (Pursain) [reply src/hooks/useRole.js:5]: eh i think its the same with the way we're using it

**Pull request merged**